app/controllers/concerns/legacy/workflow_support.rb

```diff
@@ -46,9 +46,11 @@ def create_content_blob
     private
 
     def legacy_ro_crate_params
-      params.require(:ro_crate).permit({ workflow: [:data, :data_url, :make_local_copy] },
+      l_params = params.require(:ro_crate).permit({ workflow: [:data, :data_url, :make_local_copy]},
                                        { abstract_cwl: [:data, :data_url, :make_local_copy] },
                                        { diagram: [:data, :data_url, :make_local_copy] })
+      l_params[:workflow][:project_ids] = params.dig(:workflow, :project_ids) || []
+      l_params
```

Comment:
this looks a bit weird. Can you not just add `[:project_ids]` to the workflow params, 
i.e. `.permit({ workflow: [:data, :data_url, :make_local_copy, project_ids]}`